test/system/src/test/java/io/pravega/test/system/ReadTxnWriteScaleWithFailoverTest.java

```diff
@@ -103,7 +108,7 @@ public void setup() {
         //num. of readers + num. of writers + 1 to run checkScale operation
         executorService = ExecutorServiceHelpers.newScheduledThreadPool(NUM_READERS + NUM_WRITERS + 1,
                 "ReadTxnWriteScaleWithFailoverTest-main");
-        controllerExecutorService = ExecutorServiceHelpers.newScheduledThreadPool(2,
+        controllerExecutorService = ExecutorServiceHelpers.newScheduledThreadPool(3,
```

Comment:
Why do we need to increase the number of threads? Did not understand it.

Reply:
It's not needed. It was added when I was debugging.

Reply:
reverted the change.